bindings/python/tools/python/setup.py.in

```diff
@@ -36,7 +36,8 @@ setuptools.setup(
     package_data = {
         '': [
             '${CMAKE_CURRENT_BINARY_DIR}/${NAME}/${PROJECT_PATH}/libopen-space-toolkit-physics.so.0',
-            '${CMAKE_CURRENT_BINARY_DIR}/${NAME}/${PROJECT_PATH}/${LIBRARY_TARGET}.*${EXTENSION}*.so'
+            '${CMAKE_CURRENT_BINARY_DIR}/${NAME}/${PROJECT_PATH}/${LIBRARY_TARGET}.*${EXTENSION}*.so',
+            '${CMAKE_CURRENT_BINARY_DIR}/${NAME}/${PROJECT_PATH}/test/coordinate/frame/providers/iers/data/*'
```

Comment:
remove `MANIFEST.in` as it was not taken into account by the setup.py command. Using the above line instead to ensure data are including in the distribution. Let's revisit later if needed, attempts using `MANIFEST.in` were not successful but it would indeed be cleaner!